diff --git a/node_modules/@react-native-kakao/core/android/build.gradle b/node_modules/@react-native-kakao/core/android/build.gradle
index d471e56..7d99d43 100644
--- a/node_modules/@react-native-kakao/core/android/build.gradle
+++ b/node_modules/@react-native-kakao/core/android/build.gradle
@@ -3,7 +3,7 @@ def Versions = project.ext.RNCKakaoSdkVersions
 
 buildscript {
     // Buildscript is evaluated before everything else so we can't use getExtOrDefault
-    def kotlin_version = rootProject.ext.has("kotlinVersion") ? rootProject.ext.get("kotlinVersion") : project.properties["Kakao_kotlinVersion"]
+    def kotlin_version = rootProject.ext.has("kotlin_version") ? rootProject.ext.get("kotlin_version") : project.properties["Kakao_kotlinVersion"]
 
     repositories {
         google()
